src/css/upcoming-events.css

```diff
@@ -0,0 +1,172 @@
+.events-header{
+  margin-bottom: 3rem;
+}
+.vertical-text{
+   writing-mode: vertical-rl;
+}
+.vertical{
+  letter-spacing: .24em;
+  font-size: 3.09rem;
+  /* padding-top: .5rem; */
+  text-align: left;
+}
+.r-line-overlay{
+  position: absolute;
+  z-index: 12;
+  top: 55;
+  left: 86%;
+}
+.card-border{
+  border-color: #eff0eb;
+}
+.events__bg-image{
+  top: -25%;
+  left: -5%;
+}
+
+
+.event-description{
+  line-height: 1.5rem;
+  width: 85%;
+}
+
+.event-date{
+  position: relative;
+  z-index: 30;
+  top: 0;
+  left: -13%;
+}
+
+.event-img{
+  width: 130px;
+  /* height: 160px; */
+}
+.event-img img{
+  width: 100%;
+}
+
+/* ============================== */
+.right-fund-card{
+  background-color: #d55343;
+}
+.r-card-font-color{
+  color: #ffffffbd;
+}
+.event-btn{
+  padding-top: 1rem !important;
+  padding-bottom: 1rem !important;
+  text-align: center;
+  font-size: smaller;
+}
+
+/* guage */
+.guage{
+  height: 220px;
+  width: 300px;
+}
+.c100.orange .bar, .c100.orange .fill {
+  border-color: #e6e8eb !important;
+}
+.c100.p82 .bar {
+  -moz-transform: rotate(295.2deg);
+  -ms-transform: rotate(295.2deg);
+  -webkit-transform: rotate(295.2deg);
+  transform: rotate(295.2deg);
+}
+.c100 .bar, .c100.p85 .fill{
+   position: absolute;
+  border: 0.09em solid #000000;
+  width: 0.82em;
+  height: 0.82em;
+  clip: rect(0em, 0.5em, 1em, 0em);
+  border-radius: 50%;
+  -moz-transform: rotate(0deg);
+  -ms-transform: rotate(0deg);
+  -webkit-transform: rotate(0deg);
+  transform: rotate(0deg);
+}
+.c100.p85 .bar:after, .c100.p85 .fill{
```

Comment:
Never use tailwind (or any pattern library) utility classes as selectors in your custom CSS code..either your selectors will break someone else's code (because we all use them) or you will have to increase the specificity of your selectors to avoid that which you did here. both are bad..instead I recommend making new classes and select them and keep the specificity low
you can read more about that here: 
https://css-tricks.com/strategies-keeping-css-specificity-low/
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css